pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -54,6 +56,8 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	return nil
 }
 
+var engine engineInfo
```

Comment:
Instead of defining the global variable can you define it as local variable

Reply:
Using engine locally will increase no of parameters for some functions.